File: looophole/CPP/1720.cpp
class Solution {
public:
    vector<int> decode(vector<int>& encoded, int first) {
        vector<int> ans;
        ans.push_back(first);
        int a = 0;
        for(int i=1;i<=encoded.size();i++){
            int a = ans[i-1] ^ encoded[i-1];
            ans.push_back(a);
            
        }
        return ans;
    }
};

File: looophole/Python/1720.py
class Solution:
    def decode(self, encoded: List[int], first: int) -> List[int]:
        ans = []
        ans.append(first)
        i=0
        while(i<len(encoded)):
            a = encoded[i]^ans[i]
            ans.append(a)
            i+=1
        return ans
        